### ISSUE TITLE
Why did you learning rate in optimizor is in 0?

### ISSUE THREAD

**Issue by yuchenlichuck** [graphdeco-inria/gaussian-splatting#230]
Why did you learning rate in optimizor is in 0?
       self.optimizer = torch.optim.AdamW(l, lr=0.0, eps=1e-15)# I change some how

**Comment by Snosixtyboo** [comment]
Hi,

we set the actual learning rates for each parameter group separately a few lines after the one you are referencing.

**Comment by yuchenlichuck** [comment]
ohoh thank you
